jni/src/main/java/com/horizen/schnorrnative/SchnorrPublicKey.java

```diff
@@ -16,6 +16,14 @@ private SchnorrPublicKey(long publicKeyPointer) {
     this.publicKeyPointer = publicKeyPointer;
   }
 
+  private static native FieldElement nativeGetHash();
```

Comment:
Why static ? It shouldn't be static. The CI is failing because you declared it static, but Rust side you assumed (correctly) that the method is not static, so you didn't pass the JClass object

Reply:
Done